### PR TITLE
Megaman - ZigBee Switch

### PR DESCRIPTION
**Pull request by Duncankn**

### CODE REVIEW

**Comment by shrakuma** [comment]
Hi @Duncankn
Thank you for working on integrating with our platform. We are reviewing the code and we shall keep you posted here and in the other PRs that you have submitted.
Apologies as we accidentally closed this PR.

**Comment by MAblewicz** [comment]
Reopened by Shravan's request.

File: devicetypes/smartthings/zigbee-switch.src/zigbee-switch.groovy [hunk 1]
```diff
@@ -35,7 +35,8 @@ metadata {
 		fingerprint profileId: "0104", inClusters: "0000, 0003, 0006, 0402, 0B05, FC01, FC02", outClusters: "0003, 0019", manufacturer: "iMagic by GreatStar", model: "1113-S", deviceJoinName: "Iris Smart Plug"
 		fingerprint profileId: "0104", inClusters: "0000, 0003, 0004, 0005, 0006", outClusters: "0000", manufacturer: "eWeLink", model: "SA-003-Zigbee", deviceJoinName: "eWeLink SmartPlug (SA-003)"
 		fingerprint profileId: "0104", inClusters: "0000, 0003, 0004, 0005, 0006", outClusters: "0000", manufacturer: "SONOFF", model: "BASICZBR3", deviceJoinName: "SONOFF Basic (R3 Zigbee)"
-		fingerprint profileId: "0104", inClusters: "0000, 0003, 0004, 0005, 0006, 0008, 0B05, 1000, FC82", outClusters: "000A, 0019", manufacturer: "innr", model: "SP 222", deviceJoinName: "Innr Smart Plug"
```
Comment (greens) [inline devicetypes/smartthings/zigbee-switch.src/zigbee-switch.groovy]: should not be deleting fingerprints

File: devicetypes/smartthings/zigbee-switch.src/zigbee-switch.groovy [hunk 2]
```diff
@@ -35,7 +35,8 @@ metadata {
...
+			//Megaman
+        	fingerprint profileId: "C05E", inClusters: "0000, 0003, 0004, 0005, 0006, FFFF", outClusters: "0019", manufacturer: "MEGAMAN", model: "BSZTM003", deviceJoinName: "INGENIUM® ZB Mains Switching Module"
```
Comment (tpmanley) [inline devicetypes/smartthings/zigbee-switch.src/zigbee-switch.groovy]: Please match the indentation of the existing code. It should be two tabs. Also the extra comment can be removed to follow the pattern used by the other fingerprints.

Comment (tpmanley) [reply devicetypes/smartthings/zigbee-switch.src/zigbee-switch.groovy]: This `deviceJoinName` includes `®` but the other PRs do not. I think you probably want to be consistent and I'd recommend leaving it out.

Comment (Duncankn) [reply devicetypes/smartthings/zigbee-switch.src/zigbee-switch.groovy]: @tpmanley The comment was removed and the name was modified.

Comment (Duncankn) [reply devicetypes/smartthings/zigbee-switch.src/zigbee-switch.groovy]: @tpmanley Add back the existing code.

File: devicetypes/smartthings/zigbee-switch.src/zigbee-switch.groovy (hunk 1 above)
Comment (greens) [reply devicetypes/smartthings/zigbee-switch.src/zigbee-switch.groovy]: The proper way to do this was by rebasing your branch. Copying/pasting like you've done here will make the above fingerprint seem as though you added it with this commit in the git history which makes it harder for us to determine who the original engineer was for bug-fixing purposes.

**Pull request closed without merging**